engine/action/sc_dot.cpp

```diff
@@ -743,7 +743,7 @@ void dot_t::tick()
     }
   }
 
-  sim.print_debug( "{} ticks ({} of {}).  duration={} time_to_tick={}", *this, current_tick, num_ticks(),
+  sim.print_debug( "{} ticks ({} of {}). duration={} time_to_tick={}", *this, current_tick, num_ticks(),
```

Comment:
This should have a manual sim.debug check with the new function calls.

Reply:
There are quite a few old-style debug prints in the dot code. I'd rather do them all properly in a separate PR.

Reply:
ok, makes sense not to mix in too many other changes .